.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
     rev: 22.3.0
     hooks:
     - id: black
-      language_version: python3.10
+      language_version: python3.8
```

Comment:
@AlphaGaurav7 I don't think this is a change required for this development. Can you please have it reverted?